integrations/templates/troubleshooting.md

```diff
@@ -41,6 +41,20 @@ should give you clues as to why the collector isn't working.
   ./charts.d.plugin debug 1 [[ entry.meta.module_name ]]
   ```
 
+[% elif entry.meta.plugin_name == 'xenstat.plugin' %]
+- Run the `xenstat.plugin` to debug the collector:
+
+  ```bash
+  ./xenstat.plugin debug 1 [[ entry.meta.module_name ]]
+  ```
+
+[% elif entry.meta.plugin_name == 'nfacct.plugin' %]
+- Run the `nfacct.plugin` to debug the collector:
+
+  ```bash
+  ./nfacct.plugin debug 1 [[ entry.meta.module_name ]]
+  ```
```

Comment:
I don't think xenstat and nfacct plugins have module name (`[[ entry.meta.module_name ]]`) arg. Did you check it?

Reply:
I should have worded it differently - they don't support it and @thiagoftsm didn't bother to check the source code just copy/pasted tesxt.

Reply:
That's actually true, from the README:

## Debugging

You can run the plugin by hand:

```
sudo /usr/libexec/netdata/plugins.d/xenstat.plugin 1 debug
```

You will get verbose output on what the plugin does.